students/chan/westagram/users/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db        import models
+from django.db.models import Model
+
+# Create your models here.
+class User(models.Model):
+    name                     = models.CharField(max_length=45)
+    email                    = models.EmailField(max_length=128, null=False, unique=True)
+    password                 = models.CharField(max_length=200, null=False)
+    phone_number              = models.CharField(max_length=11, null=False)
+    business_number          = models.CharField(max_length=50, blank=True)
+    is_allowing_email_recive = models.BooleanField(default=True)
+    is_enterprize            = models.BooleanField(default=True)
+    personal_site            = models.URLField(max_length=200, blank=True)
```

Comment:
```suggestion
    personal_site            = models.URLField(max_length=1000, blank=True)
```
url은 길이가 많이 길수도 있어서 1000정도 길이를 주는 것이 좋을 것 같아요.